src/udiskslinuxblock.c

```diff
@@ -1777,7 +1777,12 @@ add_remove_crypttab_entry (UDisksBlock *block,
                             &contents,
                             NULL,
                             error))
-    goto out;
+    {
+      if (g_error_matches (*error, G_FILE_ERROR, G_FILE_ERROR_NOENT))
+        contents = g_strdup ("");
```

Comment:
I guess `error` should be cleaned up then?

Reply:
Yes, good point.

Reply:
Done.

Reply:
Thanks!